<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelWidget</class>
 <widget class="QWidget" name="LabelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LabelWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_14">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#0000b3;&quot;&gt;Label&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="activeLabelLabel">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="text">
           <string>Label:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="iconSize">
           <size>
            <width>13</width>
            <height>13</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>No label</string>
           </property>
           <property name="icon">
            <iconset resource="../Main/resources/IADE_Analyzer.qrc">
             <normaloff>:/icons/transparentbox.png</normaloff>:/icons/transparentbox.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Vessel Wall</string>
           </property>
           <property name="icon">
            <iconset resource="../Main/resources/IADE_Analyzer.qrc">
             <normaloff>:/icons/bluebox.png</normaloff>:/icons/bluebox.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lumen</string>
           </property>
           <property name="icon">
            <iconset resource="../Main/resources/IADE_Analyzer.qrc">
             <normaloff>:/icons/redbox.png</normaloff>:/icons/redbox.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Calcification</string>
           </property>
           <property name="icon">
            <iconset resource="../Main/resources/IADE_Analyzer.qrc">
             <normaloff>:/icons/greenbox.png</normaloff>:/icons/greenbox.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hemorrhage</string>
           </property>
           <property name="icon">
            <iconset resource="../Main/resources/IADE_Analyzer.qrc">
             <normaloff>:/icons/yellowbox.png</normaloff>:/icons/yellowbox.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LRNC</string>
           </property>
           <property name="icon">
            <iconset resource="../Main/resources/IADE_Analyzer.qrc">
             <normaloff>:/icons/lightbluebox.png</normaloff>:/icons/lightbluebox.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LM</string>
           </property>
           <property name="icon">
            <iconset resource="../Main/resources/IADE_Analyzer.qrc">
             <normaloff>:/icons/purplebox.png</normaloff>:/icons/purplebox.png</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="text">
           <string>Label Opacity:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxOpacity">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderOpacity">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="sliderPosition">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Main/resources/IADE_Analyzer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spinBoxOpacity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderOpacity</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderOpacity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxOpacity</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
